src/librustc/session/config.rs

```diff
@@ -3273,6 +3275,10 @@ mod tests {
         opts = reference.clone();
         opts.debugging_opts.merge_functions = Some(MergeFunctions::Disabled);
         assert!(reference.dep_tracking_hash() != opts.dep_tracking_hash());
+
+        opts = reference.clone();
+        opts.debugging_opts.allow_features = Some(vec![String::from("never_type")]);
```

Comment:
Perhaps pick a feature gate that will never be stable (like one of the rustc_internal ones, maybe core_intrinsics?) to avoid confusion during stabilization?

Reply:
yeah I did that in the other test, will do